src/app/pages/vm/vm-wizard/vm-wizard.component.ts

```diff
@@ -424,7 +425,7 @@ export class VMWizardComponent implements WizardConfiguration {
           placeholder: helptext.upload_iso_path_placeholder,
           initial: '/mnt',
           tooltip: helptext.upload_iso_path_tooltip,
```

Comment:
Apply `this.translate.instant()` for all placeholders, tooltip, label

Reply:
These places don't need it as old form-control will do translations on their own. It's the new form components that need it.